src/main.ml

```diff
@@ -198,8 +185,11 @@ let cmd =
   ] @ help in
   let port =
     let doc = "Local port to listen for connections on" in
-    Arg.(value & opt int 10809 & info [ "port" ] ~doc) in
-  Term.(ret (pure main $ port)),
+    Arg.(value & opt int Consts.standard_nbd_port & info [ "port" ] ~doc) in
+  let xen_api_uri =
+    let doc = "The URI to use when making XenAPI calls. It must point to the pool master, or to xapi's local Unix domain socket, which is the default." in
+    Arg.(value & opt string Consts.xapi_unix_domain_socket_uri & info [ "xen_api_uri" ] ~doc) in
```

Comment:
I think this should be "xen-api-uri" to be more consistent with how xapi and many CLI tools name flags.

Reply:
Thanks, that sounds good, I've changed the param name.